### PR TITLE
Kg/support mix lang

### PR DESCRIPTION
**Pull request by kneth**
#2568 is labelled 0.91 but seems to be stalled. I have merged the work by @marukami with master.

Closes #2568

@realm/java

### CODE REVIEW

**Comment by stk1m1** [comment]
👍

**Comment by beeender** [comment]
@emanuelez @zaki50 need some apt gurus

**Comment by marukami** [comment]
It has been a while. I've no idea why the CI kept rejecting #2568. So,  I've just been compiling locally for the time been.

File: gradle-plugin/src/main/groovy/io/realm/gradle/Realm.groovy [hunk 1]
```diff
@@ -39,19 +39,20 @@ class Realm implements Plugin<Project> {
             throw new GradleException('Realm gradle plugin only supports android gradle plugin 1.5.0 or later.')
         }
 
-        def isKotlinProject = project.plugins.find {
-            it.getClass().name == 'org.jetbrains.kotlin.gradle.plugin.KotlinAndroidPluginWrapper'
-        }
+        def hasKotlinPlugin = project.plugins.findPlugin('kotlin-android') != null
```
Comment (emanuelez) [inline gradle-plugin/src/main/groovy/io/realm/gradle/Realm.groovy]: why is one starting with "has" and the other with "uses"? I would go with consistency here

**Comment by emanuelez** [comment]
👍 apart from minor comment but maybe @zaki50 has an opinion on how we detect the plugin

**Comment by kneth** [comment]
@marukami We did have some CI problems, and when they were fixed, we probably forgot to run the tests. I created a new branch so I could merged `master` into your work instead of asking you - sorry ;-)

**Comment by marukami** [comment]
No worries. I just hope the CI gods accept it this time :D

**Comment by zaki50** [comment]
I don't have enough time today to check the negative impact of 3d72eaebf27b2006a9c046406806bbb7dddd656e (I have a subtle concern about compatibility) .

How about reverting 3d72eaebf27b2006a9c046406806bbb7dddd656e if it's necessary?

**Comment by zaki50** [comment]
:+1:

**Pull request merged**